base/src/main/java/info/metadude/kotlin/library/c3media/models/Conference.kt

```diff
@@ -21,6 +22,8 @@ data class Conference(
         val title: String? = null,
         @Json(name = "updated_at")
         val updatedAt: OffsetDateTime? = null,
+        @Json(name = "event_last_released_at")
+        val eventLastReleasedAt: LocalDate? = null,
```

Comment:
Please extend the `ProductionApiTest` to cover the new property.

Reply:
Please note that I push a few commits on `master` meanwhile. Therefore, please rebase an force push **your** branch apply your changes on top of them.